### DISCUSSION TITLE
[TOOL] Ansible to install and maintain Azerothcore on Debian 10 (WIP)

### DISCUSSION THREAD

**Discussion by Zoidwaffle** [azerothcore/azerothcore-wotlk#3887] [Tools]
I recently started looking into Azerothcore and I personally like to do stuff using Ansible. So I made a playbook that handles it the way that I currently like and since it works for me, perhaps it could work for others as well.

You can check it out here:

[https://github.com/Zoidwaffle/azerothcore-ansible](https://github.com/Zoidwaffle/azerothcore-ansible)

It's not super polished, but it might bring some value to others so I decided to share it.

WARNING: It worked fine for me but no guarantees it will work for you - read the code if in doubt.

**Comment by Helias** [comment]
This is really interesting.
Your tool is welcome :wink:  and I think that it will be useful for other users.
Add the github topic `azerothcore-tools` to show it in the [azerothcore catalogue](https://www.azerothcore.org/catalogue/).

**Comment by Zoidwaffle** [comment]
A lot of updates has happened - some of them so big that if you already used this, stuff might have broken when updating. This can be fixed using symlinks but the easiest way is to take a backup of all your databases, and build from scratch (see the README.md, it has information regarding this option if in doubt).

If you never used this before, it should now be smoother and work far better than previous versions.

There's certainly more to work on, but I add features and fix bugs when I find out it's needed :nerd_face:
